2.4/test/run

```diff
@@ -128,10 +128,19 @@ function test_config_option() {
 -e MONGODB_USER=user
 -e MONGODB_PASSWORD=password
 -e MONGODB_ADMIN_PASSWORD=adminPassword
+-e MONGODB_SMALLFILES=1
 -e $env_var=${value//\'/}
 "
     create_container ${name} ${DOCKER_ARGS}
 
+    # need to set these because `mongo_cmd` relies on global variables
+    USER=user
+    PASS=password
+    DB=db
+
+    CONTAINER_IP=$(get_container_ip $name)
```

Comment:
In 'test_connection' there is also used function 'get_container_ip', so I suggest to remove remove this call from 'test_connection' and to use there 'CONTAINER_IP' instead of 'ip'. Or to define 'CONTAINER_IP' in 'test_connection' (change 'ip' to 'CONTAINER_IP') and not to define 'CONTAINER_IP' before calling 'test_connection'.